arch/unitroot/unitroot.py

```diff
@@ -1714,3 +1714,63 @@ def kpss_crit(stat, trend='c'):
     crit_value = interp(cv, y[::-1], x[::-1])
 
     return pvalue, crit_value
+
+  def BandWidthNW(y, kernel = "ba"):
+    '''
+    | Automatic bandwidth selection of Andrews (1991) and of Newey and West (1994)
+    | 
+    | y: Data on which to apply the bandwidth selction
+    | kernel: The kernel function to use for selecting the bandwidth
+    |  - "ba": Bartlett kernel (default value)
+    |  - "pa": Parzen kernel
+    |  - "qs": Quadratic Spectral kernel
+    | 
+    | return the Bandwidth
+    '''
+
```

Comment:
need a call to `y = np.asarray(y)` here to strip and pandas data sets.  Also need to check that `y.ndim==1` and `y` has more than `n` observations.

Reply:
Now Raise an Exception when y has not the expected properties